.github/workflows/ci-fortuna.yml

```diff
@@ -3,12 +3,16 @@ name: Check Fortuna
 on:
   pull_request:
     paths:
+      - .github/workflows/ci-fortuna.yml
       - apps/fortuna/**
   push:
     branches: [main]
 jobs:
   test:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: apps/fortuna
```

Comment:
this working-directory field was missing on a few workflows... not sure how they worked before (perhaps they didnt?)